tce/icdecoder_plugins/DefaultICDecoderPlugin.cc

```diff
@@ -1049,12 +1053,12 @@ class DefaultICDecoderGenerator : public ICDecoderGeneratorPlugin {
             *fetchBlock);
         toplevelBlock.netlist().connectPorts(*ifetchPCNextPort, *dbPCNextPort);
         NetlistPort* ifetchCyclecountPort = new NetlistPort(
-            "db_cyclecnt", "32", 32, ProGe::BIT_VECTOR, HDB::OUT,
+            "db_cyclecnt", "64", 64, ProGe::BIT_VECTOR, HDB::OUT,
             *fetchBlock);
         toplevelBlock.netlist().connectPorts(
             *ifetchCyclecountPort, *ttaCyclecountPort);
         NetlistPort* ifetchLockcountPort = new NetlistPort(
-            "db_lockcnt", "32", 32, ProGe::BIT_VECTOR, HDB::OUT,
+            "db_lockcnt", "64", 64, ProGe::BIT_VECTOR, HDB::OUT,
```

Comment:
You force 64 here, does 32 RTL generation still work? Did you run tools/scripts/compiletest.sh to check the tests? Do you have 'ghdl' installed so it runs the RTL tests?